tests/test_history.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+from banidb.history import LRUCache
+import pytest
+
+url = 'https://api.banidb.com/v2'
+target = 'mock.dat'
+cache = LRUCache(target, 25)
+
+
+@pytest.fixture()
+def put():

```

Comment:
okay to merge the PR, but this should be named something like `intialize_cache_with_shabad` instead of `put`